### ISSUE TITLE
What dictates a token's expiration time?

### ISSUE THREAD

**Issue by JakeIwen** [carlpeaslee/bouncetribe#21]
![image](https://user-images.githubusercontent.com/20415982/32210636-cac155fe-bddc-11e7-940b-efdc34911ca2.png)

![image](https://user-images.githubusercontent.com/20415982/32210652-d9d3f218-bddc-11e7-9092-f4dad3bd72bf.png)

**Comment by JakeIwen** [comment]
@carlpeaslee

**Comment by carlpeaslee** [comment]
The exp value is given to us by Auth0 with the token. I believe it is a
timecode minus the last 3 digits ( divided by 1000). It is dictated by
Auth0 there might be a way to change the length of the expiration in the
Auth0 dashboard.

Is that what you’re asking?

**Comment by JakeIwen** [comment]
Yes. I thought it was strange to see that exp * 1000, but now that makes sense.
